pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
         "PyIBIS-AMI>=4.1.2",
         "pyside2",
         "pyyaml>=6",
+        "qt>=5",
```

Comment:
Is this a conda package? I'm not sure this exists in pypi and I would have thought pyside would grab qt dependencies.  Do we need to update from pyside2 to pyside6?

Reply:
So, this one's been problematic for a while now.
I've removed it a few times over the last couple of years, for exactly the reasons you're pointing out.
But, each time I do, I get a flurry of complaints from Windows users about failed installations.
(I've got 3 or 4 of those now.)
And, sure enough, replacing it seems to always fix the problem.

Wrt/ `PySide6`, I tried to update to it a few months ago and it was a disaster.
I plan to revisit this next year, but would like to get the next release out before the end of this year.